### PR TITLE
address refactored mysql replica scripts

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@miminar fyi, fixes https://ci.openshift.redhat.com/jenkins/job/test_branch_origin_extended_image_ecosystem/257/

File: test/extended/image_ecosystem/mysql_replica.go [hunk 1]
```diff
@@ -26,19 +26,8 @@ type testCase struct {
...
-		},
-		{
-			"5.6",
-			"https://raw.githubusercontent.com/sclorg/mysql-container/master/5.6/examples/replica/mysql_replica.json",
-			false,
-		},
-		{
 			"5.7",
-			"https://raw.githubusercontent.com/sclorg/mysql-container/master/5.7/examples/replica/mysql_replica.json",
+			"https://raw.githubusercontent.com/sclorg/mysql-container/master/examples/replica/mysql_replica.json",
```
Comment (miminar) [inline test/extended/image_ecosystem/mysql_replica.go]: Why don't we use a particular commit? The repository looks alive. There will be similar errors in the future.

Comment (miminar) [reply test/extended/image_ecosystem/mysql_replica.go]: I see.

**Pull request merged**